src/index.ts

```diff
@@ -357,11 +373,20 @@ const main: JupyterFrontEndPlugin<IDebugger> = {
       }
     });
 
+    const categoryLabel = 'Debugger';
+
     if (palette) {
-      palette.addItem({ command: CommandIDs.changeMode, category: 'Debugger' });
-      palette.addItem({ command: CommandIDs.create, category: 'Debugger' });
-      palette.addItem({ command: CommandIDs.start, category: 'Debugger' });
-      palette.addItem({ command: CommandIDs.stop, category: 'Debugger' });
+      palette.addItem({
+        command: CommandIDs.changeMode,
+        category: categoryLabel
+      });
+      palette.addItem({ command: CommandIDs.create, category: categoryLabel });
```

Comment:
This could be become `palette.addItem({ command: CommandIDs.create, category });` by naming the variable `category` instead of `categoryLabel`